apps/website/src/components/Logo.module.css

```
@@ -0,0 +1,77 @@
+  align-items: center;
+  gap: 12px;
+}
+
+/* --- Sizes --- */
+.small {
+  --logo-scale: 0.8;
+}
+
+.large {
+  --logo-scale: 1.5;
+  gap: 20px;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)